docs/portable-modules/portable-modules.md

```diff
@@ -0,0 +1,54 @@
+
+# Portable Modules
+
+It is possible to build a module that targets Windows PowerShell and PowerShell Core today.
```

Comment:
I can't recommend it yet - at least not until there is a good solution around facade assemblies.